src/charts/histogramBoxPlot/onResize.js

```diff
@@ -39,6 +39,62 @@ export default function onResize() {
         const quantile = quantiles[item];
         quantile.quantile = d3quantile(this.values, quantile.probability);
 
+		// PLOT OUTLIERS
+		//--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+		if ( quantile$$1.probability == 0.05 ){
+			// probability of 0.05 corresponds to the lower whisker
+			var lower_whisker =  quantile$$1.quantile;
+			//Loop through all data points
+			for( var point in this.values ){
+				// if the data point is below the lower whisker, then it is an outlier
+				if( point < lower_whisker ){
+					console.log(point);
+					// plot as circle
+					const meanMark = this.svg
+					.append("line")
+					.attr({
+						class: "statistic",
+						x1: this.x(point),
+						y1: this.plot_height,
+						x2: this.x(point),
+						y2: this.plot_height + this.config.boxPlotHeight
+					})
+					.style({
+						fill: "#000000",
+						stroke: "black",
+						"stroke-width": "1px"
+					});
+				}
+			}
+		}
+		else if (quantile$$1.probability == 0.95){
+			// probability of 0.95 corresponds to the upper whisker
+			var upper_whisker = quantile$$1.quantile;
+			//Loop through all data points
+			console.log(upper_whisker);
+			for( var point in this.values ){
+				// if the data point is above the upper whisker, then it is an outlier
+				if( point > upper_whisker ){
+					// plot as circle
+					const meanMark = this.svg
+					.append("line")
+					.attr({
+						class: "statistic",
+						x1: this.x(point),
+						y1: this.plot_height,
```

Comment:
Try halving the height of these ticks and centering them between `this.plot_height` and `this.plot_height + this.config.boxPlotHeight`.